### ISSUE TITLE
RStudio Options keyboard shortcut doesn't work in Safari or Firefox

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#6015]
### System details

    RStudio Edition : Server
    RStudio Version : 1.2.5033, 1.3.740
    OS Version      : macOS / Safari or Firefox
    R Version       : Any

### Steps to reproduce the problem
Inside RStudio loaded in Safari or Firefox, type Cmd+comma, which is the shortcut shown for RStudio's Tools / Global Options menu item.

### Describe the problem in detail
Safari's/Firefox's preferences show instead of RStudio's.

### Describe the behavior you expected
Either we shouldn't show this as the shortcut (and perhaps have a different shortcut) when using Safari and Firefox, or it should bring up RStudio's preferences. In Chrome on macOS, it works (even though Chrome also uses Cmd+comma for preferences). Same with Edge 79.

**Comment by gtritchie** [comment]
It is possible to work around this as described here: https://apple.stackexchange.com/questions/228549/disable-cmd-in-safari

System Preferences > Keyboard > Shortcuts > App shortcuts > +
Title it "Preferences..." and remap it to Command+Alt+, and voila, you can now use Command+, in RStudio in Safari

**Comment by gtritchie** [comment]
Fixed by using Cmd+Alt+comma on Safari and FireFox on Mac, left as Cmd+comma for other browsers.

**Comment by ronblum** [comment]
Confirmed in all three browser.
